packages/api-graphql/src/internals/APIClient.ts

```diff
@@ -360,57 +360,64 @@ export function customSelectionSetToIR(
 	modelName: string,
 	selectionSet: string[]
 ): Record<string, string | object> {
-	const modelDefinition = modelDefinitions[modelName];
-	const { fields: modelFields } = modelDefinition;
+	const dotNotationToObject = (
+		path: string,
+		modelName?: string
```

Comment:
Let's make this param required. That'll resolve the build error
```suggestion
		modelName: string
```